examples/anonymous-objects/anonymous_objects.out

```diff
@@ -0,0 +1,5 @@
+# To run this sample, navigate to the directory that contains the
+# `.bal` file and use the `ballerina run` command.
```

Comment:
IINM, we've changed `use` to `issue` in other BBEs.

```suggestion
# `.bal` file and issue the `ballerina run` command.
```

Reply:
Same for `anonymous_records.out` too

Reply:
Doesn't issue sound odd?